config/software/libyaml-windows.rb

```diff
@@ -25,10 +25,13 @@
 # of rubyinstaller.org
 #
 name "libyaml-windows"
-default_version "0.1.6"
+default_version "0.1.7"
 
-source :url => "http://packages.openknapsack.org/libyaml/libyaml-0.1.6-x86-windows.tar.lzma",
-       :md5 => "8bb5d8e43cf18ec48b4751bdd0111c84",
+version "0.1.7" do
+  source sha256: "c90f4a678a7901bf0c48fc50cd6ffbd96eeb0b0184ba6c853fc0f54702213674"
+end
+
+source :url => "https://s3.amazonaws.com/dd-agent/libyaml-#{version}-x64-windows.tar.lzma",
```

Comment:
Ended up moving this to one of our buckets. Yesterday I wanted to use `dd-agent-omnibus`, which is the right place for these kinds of things, but didn't have the creds to see, much less write to the bucket. Trying to figure that out. I have a cloudops branch I'll hopefully push when I figure out what's up and what account the bucket is on.